File: .ProgramingBasics/.SimpleCalculations/11.USDToBGN/src/USDToBGN.java
import java.util.Scanner;

public class USDToBGN {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double BGN = Double.parseDouble(scanner.nextLine());
        double rate = 1.79549;
        double USD = BGN * rate;

        System.out.println(USD + " BGN");
    }
}

File: ProgramingBasics/SimpleCalculations/11.USDToBGN/11.USDToBGN/USDToBGN.cs
using System;

namespace _11.USDToBGN
{
    public class USDToBGN
    {
        public static void Main(string[] args)
        {
            var BGN = decimal.Parse(Console.ReadLine());
            var rate = (decimal)1.79549;
            var USD = BGN * rate;

            Console.WriteLine($"{USD} BGN");
        }
    }
}
